autogluon/scheduler/reporter.py

```diff
@@ -174,15 +174,29 @@ def __init__(self, process, local_reporter, dist_reporter):
 
     def run(self):
         while self.process.is_alive():
+
+            # breaking communication if process raises exception
+            if self.process.exception is not None:
+                error, traceback = self.process.exception
```

Comment:
error is not being used

Reply:
yep, since its traceback already contains all the info. I can replace it with `_`, but maybe you have other ideas how we can use it here. I was thinking maybe we can raise this error somewhere.

Reply:
If the error tells me the type of exception, it should also be transmitted. It cannot hurt, even though most generic schedulers will not be able to use it.

Reply:
so where do you think is most appropriate to use it?

Reply:
Well, normally a scheduler would just note that the evaluation failed, and it would react to any such failure in the same way. Only highly specific schedulers could do more than that.
But certainly, we should log everything we can about what happened, it may be useful.